src/components/selectionBoxLayer.ts

```diff
@@ -2,6 +2,7 @@
 
 module Plottable {
 export module Components {
+  enum PropertyMode { VALUE, PIXEL };
```

Comment:
I think that since the `PropertyMode` is not exported, `GuideLineLayer` has no access to it.  We'll need to think about it again if we want `PropertyMode` to be public.